### PR TITLE
Run daily job to send emails to suppliers about new questions and answers

### PR DESCRIPTION
**Pull request by idavidmcdonald**
This script will run everyday at 8am - https://github.com/alphagov/digitalmarketplace-scripts/pull/132

Trello: https://trello.com/c/2noI0EFB/472-3-tell-interested-suppliers-that-a-question-has-been-answered

### CODE REVIEW

File: job_definitions/notify_suppliers_of_new_questions_answers.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    scm:
+      - git:
+          url: git@github.com:alphagov/digitalmarketplace-scripts.git
+          credentials-id: github_com_and_enterprise
+          branches:
+            - master
+          wipe-workspace: false
+{% if environment == 'production' %}
+    triggers:
+      - timed: "0 8 * * *"
```
Comment (TheDoubleK) [inline job_definitions/notify_suppliers_of_new_questions_answers.yml]: This is *exactly* the same time as the "notify_suppliers_of_dos_opportunities" job.

I'd be tempted to change both jobs to `H 8 * * *` (or `H 8 * * 1-5` for the opportunities one) so they're not both kicking off together.

File: job_definitions/notify_suppliers_of_new_questions_answers.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+              URL=<${BUILD_URL}consoleFull|#${BUILD_NUMBER}>
+              CHANNEL=#dm-release
+    builders:
+      - shell: |
+          [ -d venv ] || virtualenv venv
+
+          . ./venv/bin/activate
+          pip install -r requirements.txt
+
+          if [ -n "SUPPLIER_IDS" ]; then
```
Comment (TheDoubleK) [inline job_definitions/notify_suppliers_of_new_questions_answers.yml]: Should `SUPPLIER_IDS` have a `$` in front of it here? (I don't know for sure, but suspect yes).

File: job_definitions/notify_suppliers_of_new_questions_answers.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            FLAGS="--supplier-ids=$SUPPLIER_IDS"
+          fi
+
+          if [ "$DRY_RUN" = "true" ]; then
+            FLAGS="$FLAGS --dry-run"
+          fi
+
+          ./scripts/notify-suppliers-of-new-questions-answers.py '{{ environment }}' --api-token="$DM_DATA_API_TOKEN_{{ environment|upper }}" --mandrill-api-key="$MANDRILL_API_KEY_{{ environment|upper }}" $FLAGS
```
Comment (TheDoubleK) [inline job_definitions/notify_suppliers_of_new_questions_answers.yml]: Is it too late to change the name of the script to include the word "and"? :trollface:

File: job_definitions/notify_suppliers_of_new_questions_answers.yml (hunk 2 above)
Comment (idavidmcdonald) [inline job_definitions/notify_suppliers_of_new_questions_answers.yml]: Got spot

**Comment by idavidmcdonald** [comment]
Thanks @TheDoubleK, I think we've addressed your comments. Personally I like keeping `{% set environments = ['production'] %}` even though there is only one environment, I think it's good for readability although you might argue that it's unneeded. Let me know if you feel strongly?

**Comment by TheDoubleK** [comment]
I like leaving it in there too.  Nice one :+1:

**Pull request merged**